apis/v1alpha1/tlsconfig_types.go

```diff
@@ -41,7 +41,7 @@ const (
 // - gcp: https://cloud.google.com/load-balancing/docs/use-ssl-policies#creating_an_ssl_policy_with_a_custom_profile
 // - aws: https://docs.aws.amazon.com/elasticloadbalancing/latest/application/create-https-listener.html#describe-ssl-policies
 // - azure: https://docs.microsoft.com/en-us/azure/app-service/configure-ssl-bindings#enforce-tls-1112
-type TLSConfig struct {
+type GatewayTLSConfig struct {
```

Comment:
At this point should we just move this into gateway_types.go?

Reply:
We can. It will conflict with #298